benchexec/tools/crux.py

```diff
@@ -0,0 +1,86 @@
+        return [executable] + options + list(task.input_files_or_identifier)
+
+    def version(self, executable):
+        s = self._version_from_tool(executable)
+        return s[s.find("version:"):]
+
+    def determine_result(self, run):
+        override_pat = re.compile("No implementation or override found for pointer: \"(.+?)\"")
+
+        for line in run.output:
+            # There are still a good number of functions for which Crux lacks
+            # overrides (see, for example, issue #187). Rather than reporting
+            # FALSIFIED for such programs (which will dock us points), we will
+            # instead conservatively return UNKNOWN, which doesn't lose (or
+            # gain) points. To make it more obvious which programs are UNKNOWN
+            # due to failing overrides versus incomplete goals, we include the
+            # name of the failing override in parentheses after UNKNOWN, which
+            # will show up in benchexec's reports.
+            m = override_pat.search(line)
+            if m:
+                return result.RESULT_UNKNOWN + "(no override: " + m.group(1) + ")"
+            # Crucible does not currently support inline assembly
+            elif "unsupported LLVM value: ValAsm" in line:
+                return result.RESULT_UNKNOWN + "(inline assembly)"
+            # Crucible does not currently support translating `long double`s
+            # (https://github.com/GaloisInc/crucible/issues/810)
+            elif "unsupported LLVM value: ValFP80" in line:
+                return result.RESULT_UNKNOWN + "(long double)"
+            elif "Verification result: VERIFIED" in line:
+                return result.RESULT_TRUE_PROP
+            elif "Verification result: FALSIFIED (valid-free)" in line:
+                return result.RESULT_FALSE_FREE
+            elif "Verification result: FALSIFIED (valid-deref)" in line:
+                return result.RESULT_FALSE_DEREF
+            elif "Verification result: FALSIFIED (valid-memtrack)" in line:
+                return result.RESULT_FALSE_MEMTRACK
+            elif "Verification result: FALSIFIED (valid-memcleanup)" in line:
+                return result.RESULT_FALSE_MEMCLEANUP
+            elif "Verification result: FALSIFIED (no-overflow)" in line:
+                return result.RESULT_FALSE_OVERFLOW
+            elif "Verification result: FALSIFIED (termination)" in line:
+                return result.RESULT_FALSE_TERMINATION
+            elif "Verification result: FALSIFIED (unreach-call)" in line:
+                return result.RESULT_FALSE_REACH
+            elif "Verification result: FALSIFIED" in line:
+                return result.RESULT_FALSE_PROP
+            elif "Verification result: UNKNOWN" in line:
+                return result.RESULT_UNKNOWN + "(incomplete)"
+            elif "Verification result: ERROR" in line:
+                return result.RESULT_ERROR
+        return result.RESULT_UNKNOWN
```

Comment:
Indeed, I more-or-less guessed what should go here, since I've never been able to reach this line in practice. I'm quite happy to use `result.RESULT_ERROR` here instead.